bundles/managed-api-service/1.29.0/manifests/managed-api-service.clusterserviceversion.yaml

```diff
@@ -43,16 +43,63 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     certified: "false"
-    containerImage: quay.io/integreatly/managed-api-service:master
+    containerImage: quay.io/integreatly/managed-api-service:rhoam-v1.29.0
```

Comment:
why is this image bump done?

Reply:
Reverted these changes as these were accidently included in this PR